### PR TITLE
Added option '-a' to activate the specified  plugin

### PR DESCRIPTION
**Pull request by Dos98**
-  Created an option `-a` to activate the specified plugin and disabling the rest.
   - It disables all the plugins only once 
   - `-a` can be used to activate multiple plugins

Usage: `./src/drakvuf -r /root/windows7-sp1.rekall.json -d 1 -a poolmon -a syscalls` 
This activates only the specified plugins.

- Code formatting of file is done.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -116,6 +116,7 @@
 #include "drakvuf.h"
 
 static drakvuf_c* drakvuf;
+static bool disabled_all; // Used to disable all plugin once
```
Comment (tklengyel) [inline src/main.cpp]: Please don't use global variables

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

File: src/main.cpp [hunk 2]
```diff
@@ -135,9 +135,10 @@ static inline void disable_all_plugins(bool* plugin_list)
         plugin_list[i] = false;
 }
 
-static inline void enable_plugin(char* optarg, bool* plugin_list, bool &disabled_all)
+static inline void enable_plugin(char* optarg, bool* plugin_list, bool& disabled_all)
```
Comment (tklengyel) [inline src/main.cpp]: This doesn't look right, this should be `bool *disabled_all`

File: src/main.cpp [hunk 3]
```diff
@@ -135,9 +135,10 @@ static inline void disable_all_plugins(bool* plugin_list)
...
 {
-    if(!disabled_all){
+    if (!disabled_all)
```
Comment (tklengyel) [inline src/main.cpp]: .. and this `!*disabled_all`

File: src/main.cpp (hunk 2 above)
Comment (Dos98) [reply src/main.cpp]: fixed!

**Comment by tklengyel** [comment]
Thanks, looks good!

**Pull request merged**